.github/workflows/build.yml

```diff
@@ -66,3 +67,10 @@ jobs:
           else
             echo "There were no changes to commit"
           fi
+
+      - name: Upload dist folder as artifact on commit failure
```

Comment:
Seems the build cannot even get past checking out my branch.  So this might be pointless.

Reply:
I think there might be something a little different with your environment.  I've had others create forked PRs within the last couple of weeks and they work just fine.

Reply:
My environment?  These changes I made directly from GitHub.  I did not make these changes locally and push them up.

Not, if I am a collaborator on a im-open repo (which I am on a few) and I push up a fork, it works fine.  Its when I use a fork on a repo of which I am not a collaborator.

Reply:
I would be happy to demonstrate to you how the fork is created for you to better diagnose this issue.

Reply:
This has now been resolved by adding my username as a collaborator to the repo.  Once this PR is approved, I will close it and create a new one that does not use a fork.